features/rt-cetsa-intensity-extraction-tool/pyproject.toml

```diff
@@ -0,0 +1,85 @@
+[tool.poetry]
+name = "polus_images_features_rt_cetsa_intensity_extraction"
+version = "0.2.0-dev0"
+description = "Extract well intensities from RT-CETSA plate images."
+authors = [
+    "Nick Schaub <nick.schaub@nih.gov>",
+    "Antoine Gerardin <antoine.gerardin@nih.gov>",
+    "Najib Ishaq <najib.ishaq@nih.gov>",
+]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.12"
+typer = "^0.7.0"
+filepattern = "^2.0.5"
+pandas = "^2.2.2"
+numpy = "^1.26.4"
+bfio = "^2.3.6"
+scikit-image = "^0.22.0"
+imagecodecs = "^2024.1.1"
+polus-images-segmentation-rt-cetsa-plate-extraction = {git = "https://github.com/agerardin/image-tools.git", branch = "feat/rt_cetsa", subdirectory = "segmentation/rt-cetsa-plate-extraction-tool"}
```

Comment:
This git link will need to be updated to either use the main repo from polusai and the master branch, or it should be changed to a local path dependency